README.md

```diff
@@ -1,45 +1,24 @@
 # COVIDScenarioPipeline
-Public shared code for doing scenario forecasting and creating reports for various governmental entities.
 
-# Howto
+Welcome to the Johns Hopkins University Infectious Disease Dynamics COVID-19 Working Group's `COVID Scenario Pipeline`, a flexible modeling framework that projects epidemic trajectories and healthcare impacts under different suites of interventions in order to aid in scenario planning. The model is generic enough to be applied to different spatial scales given shapefiles, population data, and COVID-19 confirmed case data. There are multiple components to the pipeline, which may be characterized as follows: 1) epidemic seeding; 2) disease transmission and non-pharmaceutical intervention scenarios; 3) calculation of health outcomes (hospital and ICU admissions and bed use, ventilator use, and deaths); and 4) summarization of model outputs.
 
-### Set up the repository
+We recommend that most new users use the code from the stable `master` branch. Please post questions to GitHub issues with the `question` tag. We are prioritizing direct support for individuals engaged in public health planning and emergency response.
 
-**Please see the [Wiki for this repository](https://github.com/HopkinsIDD/COVIDScenarioPipeline/wiki) for updated instructions on how to clone the repository and push/pull changes.**
+For more information on getting started, please visit our [Getting Started wiki](https://github.com/HopkinsIDD/COVID19_Minimal/wiki/Getting-Started) at [HopkinsIDD/COVID19_Minimal](https://github.com/HopkinsIDD/COVID19_Minimal). We are trying to keep this page up-to-date for use with the `master` branch.
 
-If making changes to this repository, please do it directly instead of through the submodule of another repository.
+This is an open-source code licensed under GPL v3.0. 
```

Comment:
Grammar is strange. Suggestion: "This open-source project is licensed under GPL v3.0."